scripts/src/java/org/oppia/android/scripts/ExemptionsList.kt

```diff
@@ -0,0 +1,617 @@
+package org.oppia.android.scripts
+
+/** Lists of files which should be exepmted for the different script checks. */
+object ExemptionsList {
+  /** List of files which should be exempted for the test file check. */
```

Comment:
```suggestion
  /**
   * List of files which should be exempted for the test file check.
   *
   * NOTE TO DEVELOPERS: New files should rarely be added here. In general, all new new files should have tests. If you think there's a valid exception, please specifically call this out in your PR description.
   */
```

The notice is important to inform developers when to change this list. The suggestion may need line wrapping.

Reply:
Done